scripts/update_meilisearch_version.sh

```diff
@@ -212,63 +211,57 @@ cp meilisearch /usr/bin/meilisearch
 systemctl restart meilisearch
 echo "${INFO_LABEL}MeiliSearch $meilisearch_version is starting."
 
-if systemctl status meilisearch | grep -E 'Active: active \(running\)' -q; then
-    echo "${INFO_LABEL}Both MeiliSearch versions are compatible. No need to dump."
-else
-    echo "${INFO_LABEL}MeiliSearch versions are not compatible with each other."
-
-    # Stopping MeiliSearch Service
-    systemctl stop meilisearch
-    echo "${INFO_LABEL}Stop MeiliSearch $meilisearch_version service."
+# Stopping MeiliSearch Service
+systemctl stop meilisearch
+echo "${INFO_LABEL}Stop MeiliSearch $meilisearch_version service."
 
-    # Keep cache of previous data.ms in case of failure
-    cp -r /var/lib/meilisearch/data.ms /tmp/
-    echo "${INFO_LABEL}Copy data.ms to be able to recover in case of failure."
+# Keep cache of previous data.ms in case of failure
+cp -r /var/lib/meilisearch/data.ms /tmp/
+echo "${INFO_LABEL}Copy data.ms to be able to recover in case of failure."
 
-    # Remove data.ms
-    rm -rf /var/lib/meilisearch/data.ms
-    echo "${INFO_LABEL}Delete current MeiliSearch's data.m."
+# Remove data.ms
+rm -rf /var/lib/meilisearch/data.ms
+echo "${INFO_LABEL}Delete current MeiliSearch's data.m."
```

Comment:
```suggestion
echo "${INFO_LABEL}Delete current MeiliSearch's data.ms"
```

Reply:
"Tomorrow the error may come from the settings being imported badly, in which case should I also check for the previous and the new settings? Errors in the migration can come from a lot of unexpected issues. Thus, I feel like the versions updates should be tested in MeiliSearch." 

It should not happen on the engine side when doing a version upgrade otherwise it is a bug.  

My suggestion is to be able to keep the speed of transferring the data.ms when possible rather than forcing the use of the dump inevitably every time. I'll make an issue! Thanks, @bidoubiwa.

Reply:
> It should not happen on the engine side when doing a version upgrade otherwise it is a bug.

Is it not the same for the indexes or data being lost?